### PR TITLE
Create chounry.json

### PR DESCRIPTION
**Pull request by chounry**
@githubteacher 
I want to building things.I want to know how they work.
This is pull request close #7660

### CODE REVIEW

**Comment by chounry** [comment]
hi

**Comment by chounry** [comment]
**i change because i want to pass**

**Comment by chounry** [comment]
{
  "state": "success",
  "target_url": "https://example.com/build/status",
  "description": "The build succeeded!",
  "context": "continuous-integration/jenkins"
}

**Comment by chounry** [comment]
hi

**Comment by chounry** [comment]
@7720njames hi

**Comment by chounry** [comment]
@AbdelhamidBouchibane  Can you help me please?

File: _pins/chounry.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-----
+githubHandle: Chounry78
```
Comment (jeffvancampen) [inline _pins/chounry.json]: Looks like the githubHandle field might be incorrect. Try using "chounry" instead of "Chounry78" ?

**Comment by hollenberry** [comment]
Almost there @chounry   ! 

Looks like you just need to remove the extra dashes. On the first and last lines, you have 5 dashes, and it should only be 3.  `---`

If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions! :tada:

**Comment by hollenberry** [comment]
@chounry  I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/7660/commits/451d3ccb5d5f35d8bf19280769fc5ce82beccca5. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by June 30th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**